ingest-app/src/cmr/ingest/services/jobs.clj

```diff
@@ -218,48 +222,72 @@
 (defn create-email-content
  "Create an email body for subscriptions"
  [from-email-address to-email-address gran-ref-location subscription]
- {:from from-email-address
-  :to to-email-address
-  :subject "Email Subscription Notification"
-  :body [{:type "text/html"
-   :content (markdown/md-to-html-string (str
-    "\nYou have subscribed to receive notifications when the following query is updated:\n\n"
-    (get-in subscription [:metadata :Query])
-    "\n\nSince this query was last run at"
-    (get-in subscription [:metadata :updated_since])
-    ", the following granules have been added or updated:\n\n"
-    (email-granule-url-list gran-ref-location)
-    "\n\nTo unsubscribe from these notifications, or if you have any questions, please contact us at [cmr-support@earthdata.nasa.gov](mailto:cmr-support@earthdata.nasa.gov).\n"
-    ))}]})
+ (let [metadata (json/parse-string (:metadata subscription))
+  concept-id (get-in subscription [:extra-fields :collection-concept-id])
+  meta-query (get metadata "Query")
+  sub-start-time (:start-time subscription)]
+  {:from from-email-address
+   :to to-email-address
+   :subject "Email Subscription Notification"
+   :body [{:type "text/html"
+    :content (markdown/md-to-html-string (str
```

Comment:
Nitpick: indentation is off here

Reply:
where do I put it?

Reply:
Ah I misread that - disregard this one :)